src/server/scripts/Northrend/IcecrownCitadel/boss_lady_deathwhisper.cpp

```diff
@@ -815,12 +791,15 @@ class npc_vengeful_shade : public CreatureScript
             npc_vengeful_shadeAI(Creature* creature) : ScriptedAI(creature)
             {
                 me->SetFlag(UNIT_FIELD_FLAGS, UNIT_FLAG_NON_ATTACKABLE);
-                me->SetFlag(UNIT_FIELD_FLAGS, UNIT_FLAG_NOT_SELECTABLE);
             }
 
             void Reset() override
             {
                 me->AddAura(SPELL_VENGEFUL_BLAST_PASSIVE, me);
```

Comment:
Vengeful Shades should stand still for a second, then follow its target. Please also check the speed with videos - I remember it being wrong.